### DISCUSSION TITLE
Responsive functionality directly in components

### DISCUSSION THREAD

**Discussion by fraktalize** [heroui-inc/heroui#717] [Feedback]
Would be great with some capacity to use responsive functionality straight in the components. Rather than overwriting with CSS, it would be great to do something like `<Text size="$2xl" mdSize="$4xl" />`

**Comment by jrgarciadev** [comment]
Hey please update to v2.0 https://nextui.org/blog/nextui-v2

You can use the TailwindCSS built-in responsive classes to do so: https://tailwindcss.com/docs/font-size#breakpoints-and-media-queries
